rs/bitcoin/adapter/BUILD.bazel

```diff
@@ -119,19 +119,26 @@ rust_test(
     data = [
         # Keep sorted.
         "//:bitcoind",
+        "//:dogecoind",
         "@bitcoin_adapter_mainnet_blocks//file",
         "@bitcoin_adapter_mainnet_headers//file",
         "@bitcoin_adapter_testnet_blocks//file",
         "@bitcoin_adapter_testnet_headers//file",
     ],
     env = {
         "BITCOIN_CORE_PATH": "$(rootpath //:bitcoind)",
+        "DOGECOIN_CORE_PATH": "$(rootpath //:dogecoind)",
         "HEADERS_DATA_PATH": "$(rootpath @bitcoin_adapter_mainnet_headers//file)",
         "BLOCKS_DATA_PATH": "$(rootpath @bitcoin_adapter_mainnet_blocks//file)",
         "TESTNET_HEADERS_DATA_PATH": "$(rootpath @bitcoin_adapter_testnet_headers//file)",
         "TESTNET_BLOCKS_DATA_PATH": "$(rootpath @bitcoin_adapter_testnet_blocks//file)",
     },
     tags = ["requires-network"],
+    # Because dogecoind binary package is not available on darwin or arm64.
+    target_compatible_with = [
+        "@platforms//cpu:x86_64",
+        "@platforms//os:linux",
+    ],
```

Comment:
The official release for dogecoind only has dmg format for OS X. We could fork it and make our own release as `tar.gz`, which then can be used by our CI. Shall we do this, or is testing on x86_86 linux enough?